kafka-client/src/test/resources/log4j.properties

```diff
@@ -17,5 +17,6 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
 log4j.logger.dev.responsive.kafka.api.async=DEBUG
+#log4j.logger.dev.responsive.kafka.internal.clients.OriginEventRecorder=DEBUG
```

Comment:
did you mean to include this?

Reply:
Yeah I did, I was using it earlier and figured I'd probably be turning it off and on. I can remove it though if it bothers our perfectionism 😆

Reply:
nope I'm good with it, just wanted to make sure!